cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/documentstore/helpers/GetFullRevisionFromCurrentCursor.java

```diff
@@ -0,0 +1,50 @@
+package com.cloudant.sync.internal.documentstore.helpers;
+
+import com.cloudant.sync.documentstore.Attachment;
+import com.cloudant.sync.internal.documentstore.DocumentBodyImpl;
+import com.cloudant.sync.internal.documentstore.DocumentRevisionBuilder;
+import com.cloudant.sync.internal.documentstore.InternalDocumentRevision;
+import com.cloudant.sync.internal.sqlite.Cursor;
+
+import java.util.Map;
+
+/**
+ * Created by tomblench on 23/08/2017.
+ */
+
+public class GetFullRevisionFromCurrentCursor {
+
+    public static InternalDocumentRevision call(Cursor cursor,
```

Comment:
It's a bit confusing that these use the `call` method name, but aren't `Callable` instances. That said I'm not sure I can think of a better naming pattern, other than to make the methods follow the class names.